pages/index.vue

```diff
@@ -1,4 +1,57 @@
+<script lang="ts" setup>
+// Mock data, to be replaced with real data from the API
+const statuses = ['todo', 'doing', 'done']
+
+const issues = [
```

Comment:
I've explained why it would be good to have a mock data schema that matches httpd's response. But that's fine here for now since we've patched it on your follow-up PR.